docs/TutorialWebpack.md

```diff
@@ -158,9 +158,32 @@ of the `moduleNameMapper` option again.
 }
 ```
 
-That's it! Webpack is a complex and flexible tool, so you may have to make some adjustments
+That's it! webpack is a complex and flexible tool, so you may have to make some adjustments
 to handle your specific application's needs. Luckily for most projects, Jest should be more than
-flexible enough to handle your Webpack config.
+flexible enough to handle your webpack config.
 
-*Note: For more complex Webpack configurations, you may also want to investigate
+*Note: For more complex webpack configurations, you may also want to investigate
 projects such as: [babel-plugin-webpack-loaders](https://github.com/istarkov/babel-plugin-webpack-loaders).*
+
+
+## webpack 2
+
+webpack 2 beta offers native support for ES modules. However, Jest runs in
```

Comment:
let's drop the beta here. I suppose this won't change in the final and I don't want to have to update the docs later to remove it :)